gio/src/output_stream.rs

```diff
@@ -65,6 +65,79 @@ pub trait OutputStreamExtManual: Sized + OutputStreamExt {
         >,
     >;
 
+    #[cfg(any(feature = "v2_60", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_60")))]
+    #[doc(alias = "g_output_stream_writev")]
+    fn writev(
+        &self,
+        vectors: &[&[u8]],
```

Comment:
This wouldn't allow to use a `Vec<Vec<u8>>` for example. Should be made a bit more generic at least

Reply:
Same applies to other functions here

Reply:
Changed to use `OutputVector` directly. It is not more generic but it avoids allocations. The other option is to keep the allocation and make it `IntoIterator<Item = &dyn AsRef<[u8]>>`

Reply:
I think this is all set the way it is, any other API will require allocations. This way the caller can choose whether to allocate or use the stack.